README.md

```diff
@@ -29,16 +29,16 @@ For more detailed information on getting the best visual quality and performance
 
 # OpenXR preview extensions
 
-The [openxr_preview](https://github.com/microsoft/OpenXR-MixedReality/tree/master/openxr_preview) folder contains a set of [preview header files](https://github.com/microsoft/OpenXR-MixedReality/tree/master/openxr_preview/include/openxr) containing the following preview OpenXR extensions:
+The [openxr_preview](https://github.com/microsoft/OpenXR-MixedReality/tree/master/openxr_preview) folder contains a set of [preview header files](https://github.com/microsoft/OpenXR-MixedReality/tree/master/openxr_preview/include/openxr) containing the following OpenXR extensions that's only available [in preview runtime](http://aka.ms/openxr-preview).

```

Comment:
(...) containing the following OpenXR extensions **that are** only available [in **the** preview runtime](http://aka.ms/openxr-preview).